frontend/src/pages/Contact.jsx

```
@@ -0,0 +1,368 @@
+                        </div>
+                        <textarea
+                          id="message"
+                          name="message"
+                          value={formData.message}
+                          onChange={handleInputChange}
+                          rows="5"
+                          className="w-full pl-10 pr-3 py-3 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent"
+                        //   placeholder={showVendorForm ? "Tell us about your property and business..." : "How can we help you?"}
+                          placeholder={"Tell us about your property and business..."}
+                          required
+                        ></textarea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)